test/models/binomial.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+from scipy import stats
+from scipy.special import logit, expit as inv_logit, log1p
+import numpy as np
+import numpy.typing as npt
+
+
+class Binomial:
+    """
+    Binomial model with a conjugate beta prior.
+
+    This model's joint density is factored as the product of
+    the prior density and the likelihood for testing with
+    samplers that require that (e.g., SMC).
```

Comment:
Honestly, might be better to note this next to the log_density function, anyway--at least, that's where I was looking when I got confused about why it was broken into the two component parts. But I don't want to bikeshed too much here

Reply:
Agreed.  The only reason to be more picky on these first few is to establish conventions going forward.  Talking about log densities rather than densities is problematic because there's no equivalent word to "factored" and there's no equivalent symbol to propto but meaning up to an additive constant.  The log scale needs more lexical love!